docs/api/wrapper/setChecked.md

```diff
@@ -1,9 +1,19 @@
-## setChecked(value)
+## setChecked(checked)
 
-Sets the value of a radio or checkbox `<input>`.
+Sets the value to an input element of type checkbox or an input element of type radio.
+
+When you try to set the value to state via `v-model` by `radioInput.element.checked = true; radioInput.trigger('input')`, `v-model` is not triggered. `v-model` is triggered by `change` event.
```

Comment:
I think we should add this to a `notes` sections after the arguments and example, I don't think most users need this info